nnlorun.py

```diff
@@ -344,23 +345,51 @@ def copy_to_grid(local_file, grid_file, args, maxrange=MAX_COPY_TRIES):
 
 
 def grid_copy(infile, outfile, args, maxrange=MAX_COPY_TRIES):
```

Comment:
The new function looks good, a bit saver then the old. Could you please add them to `hejrun.py` and `sherparun.py'? We should probably keep these basic functions synchronised between the different scripts

Reply:
Will do (once we've settled on the final version).  Commit with checksum coming soonish